rules/impl.bzl

```diff
@@ -116,6 +117,13 @@ def _run_android_lint(
     args.add("--output", output)
     outputs.append(output)
 
+    args.add("--html-output", html_output)
+    outputs.append(html_output)
+
+    label = ctx.attr.android_home.label
+    if ctx.attr.android_home:
```

Comment:
This doesn't look like its intended to be in this PR

Reply:
D'oh good catch